module-4/backend/week_one.md

```diff
@@ -5,17 +5,30 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+The most useful thing I learned over intermission was how to separate functions and then call them from each other. 
 2. What are some tools to help debug JavaScript code?
+Debugger, alert, console.log, pryjs. 
 3. What are some tools you need in order to unit test your JavaScript?
+Mocha, Chai
 4. What is the syntax for invoking a function?
+  functionName(param)
 5. What is CORS?
+  Cross Origin Resource Sharing
 6. How do we enable CORS?
+  I beleive that when you are making an ajax call, you can set the mode to CORS as one of the headers of the request. 
 7. What's `this` in JavaScript?
+  This is contextual identifier based on your scope. 
 8. What is Webpack and why is it useful?
+  Webpack is a tool to consolidate all of your javascript files. It also performs useful functions such as transcompiling and minifying your code. 
 9. When/why do you want to use event delegation?
+  When you have a specific event that needs to occure as part of another event. 
```

Comment:
Not quite that is a better definition for event bubbling - this is more for attaching event listeners to things that may have not been added to the DOM yet (items on a to-do list, for example)